beginnercodes/challenges/test_runner.py

```diff
@@ -23,17 +29,41 @@ class Result:
     status: StatusEnum = field(default=StatusEnum.SUCCESS)
 
 
-def test(challenge: int, solution_func):
+def build_table():
+    _t = Table(title='[bold]Test Results[/bold]', highlight=True, box=box.MINIMAL)
+    _t.add_column("#:", justify='right')
+    _t.add_column("Expected:", justify='center')
+    _t.add_column("Got:")
+    return _t
+
+
+def test(challenge: int, solution_func, description: bool = False, examples: bool = False):
+    console = Console(theme=Theme({'numbers': 'blue', 'expected': 'green', 'got': 'red', 'final': 'yellow'}))
+
     ### These functions must be protected to ensure no CHEATING happens. ###
-    def _get_tests(challenge: int) -> list[dict]:
-        response = requests.get(
-            f"https://raw.githubusercontent.com/beginner-codes/challenges/main/weekday/test_cases_{challenge}.json"
-        )
-        if response.status_code == 404:
-            raise ChallengeNotFound(f"Challenge {challenge} was not found.")
+    def _fetch_data(url:str) -> Union[Response, str]:
+        _r = requests.get(url.replace('XXXX', str(challenge)))
+        if _r.status_code == 404:
+            raise ChallengeNotFound(f"Challenge {challenge} is not found")
+        return _r
 
+    def _get_tests(challenge: int) -> list[dict]:
+        url = "https://raw.githubusercontent.com/beginner-codes/challenges/main/weekday/test_cases_XXXX.json"
+        response = _fetch_data(url)
         return response.json()
 
+    def _get_info(challenge: int, description: bool, examples: bool) -> str:
+        url = "https://raw.githubusercontent.com/beginner-codes/challenges/main/weekday/challenge_XXXX.md"
+        result = ''
+        if description or examples:
+            info = _fetch_data(url)
+            try:
+                result += info.text.split('##')[0] if description else ''
+                result += '# ' + info.text.split('# ')[2].replace('#','') if examples else ''
```

Comment:
Don't use a ternary if's here. Use `if description` and `elif examples`. Simpler, easier to understand, and results in less op codes to be run.